### PR TITLE
Docs(plugin) : Add ContextExclusionPlugin

### PR DESCRIPTION
**Pull request by jeffin143**
In reference with  :
https://github.com/webpack/webpack/blob/855241ad3a155df01b2d24df31efc1785880870d/lib/ContextExclusionPlugin.js#L32

- [x] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [x] Make sure your PR complies with the [writer's guide][2].
- [x] Review the diff carefully as sometimes this can reveal issues.
- [x] Do not abandon your Pull Request: [Stale Pull Requests][3].

### CODE REVIEW

File: src/content/plugins/context-exclusion-plugin.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: ContextExclusionPlugin
+contributors:
+  - jeffin
+---
+
+The `ContextExclusionPlugin` allows you to exclude some of the context. A regex could be provided to plugin to configure it to exlude all context that matches it.
```
Comment (montogeek) [inline src/content/plugins/context-exclusion-plugin.md]:
```suggestion
The `ContextExclusionPlugin` allows you to exclude some of the context. A regex could be provided to the plugin to configure it to exlude all context that matches it.
```

File: src/content/plugins/context-exclusion-plugin.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: ContextExclusionPlugin
+contributors:
+  - jeffin
+---
+
+The `ContextExclusionPlugin` allows you to exclude some of the context. A regex could be provided to the plugin to configure it to exlude all context that matches it.
```
Comment (jamesgeorge007) [inline src/content/plugins/context-exclusion-plugin.md]:
```suggestion
The `ContextExclusionPlugin` allows you to exclude some of the context. A regex could be provided to the plugin to configure it to exclude all context that matches it.
```

**Comment by jeffin143** [comment]
> Be specific: it allows to exclude (what? resources? paths?) based on a given RegExp

it exclude all context module based on the given RegExp , some is realtive word that means if given it would exclude all of those context (some since there would be many context -> so some among all the given context)

File: src/content/plugins/context-exclusion-plugin.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+title: ContextExclusionPlugin
+contributors:
+  - jeffin
+---
+
+_Context_ refers to a [require with an expression](/guides/dependency-management/#require-with-expression) such as `require('./locale/' + name + '.json')`.
+
+The `ContextExclusionPlugin` allows you to exclude context. A regex could be provided to plugin to configure it to exclude all context that matches it.
```
Comment (EugeneHlushko) [inline src/content/plugins/context-exclusion-plugin.md:9]:
```suggestion
The `ContextExclusionPlugin` allows you to exclude context. Provide RegExp as an argument when initializing the Plugin to exclude all context that matches it.
```

File: src/content/plugins/context-exclusion-plugin.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+__webpack.config.js__
+
+``` javascript
+module.exports = {
+	plugins: [
+        new webpack.ContextExclusionPlugin(/dont/)
+    ]
+};
+
```
Comment (EugeneHlushko) [inline src/content/plugins/context-exclusion-plugin.md:19]:
```suggestion
  plugins: [
    new webpack.ContextExclusionPlugin(/dont/)
  ]
};
```

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**